includes/module/preference/AbstractPreference.php

```diff
@@ -348,15 +348,16 @@ public function getCustomCustomerData($cart)
      */
     public function getShipmentAddress($cart)
     {
-        $address_invoice = new Address((int) $cart->id_address_invoice);
+        $address_shipment = null;
+        ($this->use_same_address)? $address_shipment = new Address((int) $cart->id_address_invoice) : $address_shipment = new Address((int) $cart->id_address_delivery);
```

Comment:
De onde vem essa propriedade `$this->use_same_address` ?

Reply:
Tem um checkbox com essa variável, Gio.